tests/readers/opendss/Capacitors/test_capacitor_kvar.dss

```diff
@@ -0,0 +1,11 @@
+Clear
+
+New circuit.test_capacitor_kvar basekv=4.16 pu=1.01 phases=3 bus1=sourcebus
+
+New Capacitor.Cap1 Bus1=bus1 phases=3 kVAR=600 kV=4.16
```

Comment:
I don't know anything about the opendss format, but is it possible to have an unbalanced capacitor? if so, we should probably test that case

Reply:
@etimberg I think you'd model that as multiple one phase capacitors rather than one unbalanced capacitor (I'll have to double check). 
I agree that these tests are probably not complete but this provides examples to get started for Bhavya who is going to work on this really soon.